src/components/Books.js

```diff
@@ -0,0 +1,12 @@
+import '../App.css';
+import Book from './Book';
+
+const Books = () => (
+  <div>
+    <Book title="The Hunger Games1" author="Suzanne Collins1" />
+    <Book title="The Hunger Games1" author="Suzanne Collins1" />
+    <Book title="The Hunger Games1" author="Suzanne Collins1" />
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] I would recommend calling the `Input` component from here(`Books`) component rather than `App.js`. By this, it would maintain the hierarchy and the requirement as well.

Reply:
@Dipeshtwis  Added successfully
![e58](https://user-images.githubusercontent.com/40554384/166067919-af91477f-a087-4cb3-93e8-75fa4c82144a.PNG)